qiskit/providers/models/pulsedefaults.py

```diff
@@ -94,44 +101,325 @@ class Command(BaseModel):
+            buffer: Default buffer time (in units of dt) between pulses.
+            pulse_library: Pulse name and sample definitions.
+            cmd_def: Operation name and definition in terms of Commands.
+            **kwargs: Other attributes for the super class.
+        """
+        super().__init__(**kwargs)
+
         self.buffer = buffer
+        self._qubit_freq_est_ghz = qubit_freq_est
+        self._meas_freq_est_ghz = meas_freq_est
+        self._qubit_freq_est_hz = [freq * 1e9 for freq in qubit_freq_est]
+        self._meas_freq_est_hz = [freq * 1e9 for freq in meas_freq_est]
+        # TODO: These should be massaged for the user
         self.pulse_library = pulse_library
         self.cmd_def = cmd_def
 
-        super().__init__(**kwargs)
+        # The processed and reformatted circuit operation definitions
+        self._ops_def = defaultdict(dict)
+        # A backwards mapping from qubit to supported operation
+        self._qubit_ops = defaultdict(list)
+
+        # Build the above dictionaries from pulse_library and cmd_def
+        self.converter = QobjToInstructionConverter(pulse_library, buffer)
+        for op in cmd_def:
+            qubits = _to_tuple(op.qubits)
+            self._qubit_ops[qubits].append(op.name)
+            pulse_insts = [self.converter(inst) for inst in op.sequence]
+            self._ops_def[op.name][qubits] = ParameterizedSchedule(*pulse_insts, name=op.name)
+
+    @property
+    def qubit_freq_est(self) -> List[float]:
+        """
+        Return the estimated resonant frequency for the given qubit in Hz.
+
+        Returns:
+            The frequency of the qubit resonance in Hz.
+        """
+        warnings.warn("The qubit frequency estimation was previously in GHz, and now is in Hz.")
+        return self._qubit_freq_est_hz
+
+    @property
+    def meas_freq_est(self) -> List[float]:
+        """
+        Return the estimated measurement stimulus frequency to readout from the given qubit.
+
+        Returns:
+            The measurement stimulus frequency in Hz.
+        """
+        warnings.warn("The measurement frequency estimation was previously in GHz, "
+                      "and now is in Hz.")
+        return self._meas_freq_est_hz
+
+    def ops(self) -> List[str]:
+        """
+        Return all operations which are defined. By default, these are typically the basis gates
+        along with other operations such as measure and reset.
+
+        Returns:
+            The names of all the circuit operations which have Schedule definitions in this.
+        """
+        return list(self._ops_def.keys())
 
-    def build_cmd_def(self):
-        """Construct the `CmdDef` object for the backend.
+    def op_qubits(self, operation: str) -> List[Union[int, Tuple[int]]]:
+        """
+        Return a list of the qubits for which the given operation is defined. Single qubit
+        operations return a flat list, and multiqubit operations return a list of ordered tuples.
```

Comment:
`op_qubits` is almost too short of a name. Can we have a name that has more meaning? The first thing that comes to mind is `qubits_with_operation` but that almost seems too wordy (or maybe `qubits_with_op`). Suggestions?
```
cx_qubits = qubits_with_op('cx')
```

Reply:
I was trying to be a bit consistent with `cmd_qubits`. I will definitely put some thought into this.

Reply:
I took the suggestion, thank you @eggerdj